appstore/api/error.go

```diff
@@ -23,6 +29,26 @@ type appStoreAPIErrorResp struct {
 	ErrorMessage string `json:"errorMessage"`
 }
 
+func newAppStoreAPIError(b []byte, reqHeader http.Header) (*Error, bool) {
```

Comment:
I know it's a detail, but does the name here mean `respHeader` instead of `reqHeader`?
Or simply `hd`, etc. would be fine ☺️

Reply:
@takecy , Sorry for the typo, I prefer the `hd`, and the fix is done. Could you please review it again?